lib/usb/usb_process.js

```diff
@@ -13,47 +11,46 @@ function USBProcess(id, daemon) {
   var self = this;
   this.id = id;
   // 4 remote process streams
-  this.control = new stream.Writable;
+  this.control = new stream.Writable();
   this.control.credit = 0;
   this.control.backPressure = new Array(0);
-  this.stdin = new stream.Writable;
+  this.stdin = new stream.Writable();
   this.stdin.credit = 0;
   this.stdin.backPressure = new Array(0);
-  this.stdout = new stream.Readable;
+  this.stdout = new stream.Readable();
   this.stdout.credit = 0;
-  this.stderr = new stream.Readable;
+  this.stderr = new stream.Readable();
   this.stderr.credit = 0;
   // Status of whether this process is killed or active
   this.active = true;
   // Code to be set upon death
-  this.exitCode;
-
-  var self = this;
+  this.exitCode = null;
 
   this.stdout._read = function() {};
   this.stderr._read = function() {};
 
-  // This puts the stream into 'flowing mode' 
+  // This puts the stream into 'flowing mode'
   // so that the stream emits ('end') events without 'data' listeners
   this.stdout.resume();
   this.stderr.resume();
 
   this.stdout.signal = function(signalToSend) {
-    switch(signalToSend) {
+    // RWQ: Does this need to a `switch`?
```

Comment:
It eventually will; I only have one case at the moment but I suspect there will be more signals that we'd like to send. In fact, #126 adds another case.